packages/create-remix/templates/cloudflare-pages/functions/[[path]].js

```diff
@@ -0,0 +1,15 @@
+import { createRequestHandler } from "@remix-run/cloudflare-pages";
+
+// @ts-ignore
+import * as build from "../build";
+
+const handleFetch = createRequestHandler({
+  build
+});
+
+export function onRequest(context) {
+  const request = new Request(context.request);
+  request.headers.delete("If-None-Match");
```

Comment:
We'll make sure this works before GA. Sorry!

Reply:
Just made one [over here](https://github.com/cloudflare/wrangler2/issues/117). Pretty certain this is something on our side, but not exactly sure where, so we'll take a look and circle back once we know what's going on.